### PR TITLE
fix(e2e): Fix the get-chrome-url function

### PR DESCRIPTION
**Pull request by ioanpocol**

### CODE REVIEW

**Comment by sivakuna-aap** [comment]
:+1:

**Comment by amagdas** [comment]
:+1:

**Comment by pavlovicnemanja** [comment]
:+1:

File: scripts/get-chrome-url [hunk 1]
```diff
@@ -12,10 +12,15 @@ payload = {'os': OS, 'channel': CHANNEL}
...
 if build == 'unknown':
 	# The latest passing build
-	build = '338390'
+	build = '353893'
 
 print(BUILDS_URL.format(build))
 
-cd 
\ No newline at end of file
+cd
```
Comment (petrjasek) [inline scripts/get-chrome-url]: imo there should be no `cd`, it throws exception

**Pull request merged**